test/helpers/describe_installer.js

```diff
@@ -40,7 +40,7 @@ module.exports.cleanupOutputDir = function cleanupOutputDir (outputDir) {
 }
 
 module.exports.tempOutputDir = function tempOutputDir (customDir) {
-  return customDir ? path.join(os.tmpdir(), customDir) : temp.path({ prefix: 'electron-installer-debian-' })
+  return customDir ? path.join(os.tmpdir(), customDir) : tmp.tmpNameSync({ prefix: 'electron-installer-debian-' })
 }
```

Comment:
I'm also considering adding a few of the exported functions in here into `electron-installer-common/src/testhelper` (not re-exported in `src/index.js`).